falcon/response.py

```diff
@@ -499,10 +499,11 @@ def add_link(self, target, rel, title=None, title_star=None,
         'Content-Range',
         """A tuple to use in constructing a value for the Content-Range header.
 
-        The tuple has the form (*start*, *end*, *length*), where *start* and
-        *end* designate the byte range (inclusive), and *length* is the
-        total number of bytes, or '\*' if unknown. You may pass ``int``'s for
-        these numbers (no need to convert to ``str`` beforehand).
+        The tuple has the form (*start*, *end*, *length*, [*unit*]), where *start* and
+        *end* designate the range (inclusive), and *length* is the
+        total length, or '\*' if unknown. You may pass ``int``'s for
+        these numbers (no need to convert to ``str`` beforehand). Optional value
```

Comment:
"Optional value *unit* describes..." ==> "The optional *unit* value describes..."